reporter/lib/events.js

```diff
@@ -6,39 +6,39 @@ module.exports = (socket, id) => {
                 return;
             }
 
-            socket.emit('control:new-run', {
+            socket.emit('start', {
```

Comment:
Still using string and not constants... 🤔

Reply:
I mentioned this [here](https://github.com/elit-altum/newman-dashboard/pull/9#discussion_r660448245) in the last PR. I will move them together after we find all the event names and listeners in Newman! 
I will move towards this today

Reply:
Made a new ticket https://github.com/elit-altum/newman-dashboard/issues/24